ui/dashboards/src/components/AddPanel/AddPanel.tsx

```diff
@@ -41,26 +44,42 @@ const AddPanel = ({ isOpen, onClose }: AddPanelProps) => {
   const [group, setGroup] = useState(0);
   const [panelName, setPanelName] = useState('');
   const [panelDescription, setPanelDescription] = useState('');
+  const [kind, setKind] = useState('');
+  const [options, setOptions] = useState<JsonObject>({});
 
-  const onSelectGroupChange = (e: SelectChangeEvent<number>) => {
-    setGroup(e.target.value as number);
+  const handleGroupChange: SelectProps<number>['onChange'] = (e) => {
+    const { value } = e.target;
+
+    // Handle string (which would be empty string but shouldn't happen since we don't allow a "None" option) by
+    // just ignoring it
+    if (typeof value === 'string') return;
```

Comment:
Oh I see, if it's non-empty, does the typing above convert it to a `number`?

Reply:
Yeah, the typing there knows value is `string | number` from MUI's prop definitions.